snippets/language-html.cson

```diff
@@ -1,4 +1,4 @@
-'.text.html, .text.html .meta.scope.between-tag-pair, .text.html .punctuation.tag.begin':
+'.text.html, .text.html .meta.scope.between-tag-pair, .text.html .punctuation.tag.begin, .source.gfm':
```

Comment:
That sounds more reasonable, however the best change would probably be to add those snippets to language-gfm's snippets list.

(Not sure if that's what you mean)